packages/lab/src/__tests__/__e2e__/date-picker/DatePicker.cy.tsx

```diff
@@ -121,6 +121,59 @@ describe("GIVEN a DatePicker", () => {
       cy.findAllByRole("application").should("exist");
     });
   });
+
+  describe("WHEN range datepicker is mounted with visibleMonths 1", () => {
```

Comment:
When I originally asked I meant testing that only one calendar was visible, but adding these tests too is fine

Reply:
Ah I see it's on #162 but can we make it a standalone test so it's clear that's what we're testing

Reply:
But we still need one for the button